packages/server/src/connection/server.ts

```diff
@@ -256,6 +264,18 @@ export class Server extends core.EventLogEmitter {
             const actionProto = await proto.ActionProto.importProto(decryptedMessage);
 
             return new Promise<proto.ResultProto>((resolve, reject) => {
+              const handleResolve = (reason: proto.ResultProto) => {
+                if (this.options.onMessageResult) {
+                  this.options.onMessageResult(true);
```

Comment:
I'm not sure it's correct. What if `onMessageResult` throws exception, especially in a case of `Promise`